addons/io_scene_gltf2/blender/imp/gltf2_blender_material_utils.py

```diff
@@ -93,3 +100,65 @@ def make_texture_block(gltf, node_tree, tex_info, location, label, name=None, co
     node_tree.links.new(tex_img.inputs[0], mapping.outputs[0])
 
     return tex_img
+
+NEAREST = 9728
+LINEAR = 9729
+NEAREST_MIPMAP_NEAREST = 9984
+LINEAR_MIPMAP_NEAREST = 9985
+NEAREST_MIPMAP_LINEAR = 9986
+LINEAR_MIPMAP_LINEAR = 9987
```

Comment:
We should centralized all the constants in a same place, for example here, where some are already listed:
io/com/gltf2_io_constants.py

Reply:
I added a commit. Is that what you had in mind?

Reply:
Yes, it is :)